src/app/src/mainwindow.cpp

```diff
@@ -907,6 +912,15 @@ void MainWindow::setupNIDev(std::string niDev) {
 
     m_DAQmx.CreateAnalogOutpuVoltageChan(m_taskAO, m_devAO.c_str(), -10.0, 10.0, DAQmx_Val_Volts);
     m_DAQmx.CreateDigitalOutputChan(m_taskDO, m_devDO.c_str(), DAQmx_Val_ChanForAllLines);
+
+    //Setup NIDAQmx controller for manual trigger
+    m_taskDO_2 = "Digital_Out_2";
+    m_devDO_2 = fmt::format("{}/ao0", m_config->orDev);
```

Comment:
I think this is the wrong format for the digital out, it should be something like
`m_devDO = fmt::format("{}/port0/line0:7", m_config->niDev); //Device for digital output`